integration/docker/entrypoint.sh

```diff
@@ -52,6 +52,7 @@ case ${service,,} in
     integration/docker/bin/alluxio-master.sh
     ;;
   worker)
+    export ALLUXIO_RAM_FOLDER=${ALLUXIO_RAM_FOLDER:-/dev/shm}
```

Comment:
`ALLUXIO_RAM_FOLDER` isn't a site property - it's only used in `alluxio-config.sh` to configure the tier 0 directory. It's in the list of exclusions so that we don't translate it into alluxio-site.properties.

I don't think any behavior is affected by moving the export earlier in the file